test/playbooks_reader_test.rb

```diff
@@ -55,7 +55,7 @@ class PlaybooksReaderTest < Minitest::Test
 
   describe 'playbooks_names method' do
     it 'should return playbooks names as an array' do
-      playbooks_names = %w[xprazak2.forklift_collection.freeipa_server.yml xprazak2.forklift_collection.upgrade.yml]
```

Comment:
Could we keep the `.yml` here as that is what the Dir.glob should return and validate that the result doesn't have these?
Ideally we would validate the correct names.

Reply:
yes, sorry it was a mistake